invenio_app_ils/items/mappings/v6/items/item-v1.0.0.json

```diff
@@ -178,18 +189,6 @@
                 "name": {
                   "type": "keyword"
                 },
-                "address": {
```

Comment:
for the reviewers: I requested the change, since it is highly unlikely that we will search items using email of the location and the other removed fields

Reply:
Ok, but then we should change the resolvers too? This is only removing the mappings. The will be still indexed, but ES will auto-guess the type.

Reply:
true,  we shouldn't resolve those fields IMO